pkgs/development/python-modules/image-match/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, pytestrunner, scikitimage, six }:
+
+buildPythonPackage {
+  pname = "image-match";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
+    owner = "ascribe";
+    repo = "image-match";
+    rev = "1c5f3170555540bdf43ff8b8189c4e8c13a8b950";
+    sha256 = "0vlmpidmhkpgdzw2k03x5layhijcrjpmyfd93yv2ls77ihz00ix5";
+  };
+
+  buildInputs = [ pytestrunner ];
```

Comment:
checkInputs

Reply:
The `pytestrunner` is part of the `setup_requires`, I thought that meant it was required regardless of tests.

Reply:
You're correct, `buildInputs` it is.

Looking at their `setup.py` there seems to be no need for having it in `setup_requires`.